### PR TITLE
Add new field for docker image

### PR DESCRIPTION
**Pull request by Jan-M**
Add new field "source_base"

### CODE REVIEW

File: zmon_aws_agent/aws.py [hunk 1]
```diff
@@ -222,6 +222,7 @@ def get_running_apps(region):
                     ins['application_version'] = user_data['application_version']
 
                 ins['source'] = user_data['source']
+                ins['source_base'] = "".join(ins['source'].split(":")[:1])
```
Comment (mohabusama) [inline zmon_aws_agent/aws.py]: Can't this just be ``.split(':')[0]`` ?

Comment (Jan-M) [inline zmon_aws_agent/aws.py]: henning says this is the trick for no match

**Comment by Jan-M** [comment]
:+1:

**Comment by Jan-M** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Comment by hjacobs** [comment]
Tests broken..

**Comment by Jan-M** [comment]
:+1:

**Comment by Jan-M** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**